src/Query/Query.ts

```diff
@@ -24,7 +25,7 @@ export class Query implements IQuery {
     private readonly shortModeRegexp = /^short/;
     private readonly explainQueryRegexp = /^explain/;
 
-    private readonly limitRegexp = /^limit (to )?(\d+)( tasks?)?/;
+    private readonly limitRegexp = /^limit (group )?(to )?(\d+)( tasks?)?/;
```

Comment:
I did some smoke tests and it worked well.

### Thinking out loud

The only thing is that typing any of these didn't read right to me...

~~~
limit group 7
limit group to 7 tasks
~~~

It's interesting to me that in the description of the PR you wrote:

> Add `limit groups <number>` & `limit group to <number> tasks` instructions.

I know in #1674 we all said group singular, but that was before I typed it out in actual queries.

I feel that the following reads better:

~~~
limit groups to 7 tasks
~~~

I am not sure between these two:

~~~
limit group 7
limit groups 7
~~~

### Request

Because this:

~~~
limit groups to 7 tasks
~~~

... reads so much better than this:

~~~
limit group to 7 tasks
~~~

I recommend renaming the new instructions to:

~~~
limit groups 7
limit groups to 7 tasks
~~~

And in examples, writing the instruction out in full::

```
limit groups to 7 tasks
```

Reply:
This does sound better. Fixed

Reply:
Thanks! You're probably working on it, but if not, please do update the docs to match the new spelling.

Reply:
Oops, I actually forgot that. Thanks for your reminder! =)